tests/unit/worker/job_poller_test.py

```diff
@@ -105,3 +101,22 @@ def test_calculate_max_jobs_to_activate(self, job_poller: JobPoller, active_jobs
         max_jobs_to_activate = job_poller.calculate_max_jobs_to_activate()
 
         assert max_jobs_to_activate == expected
+
+
+@pytest.mark.asyncio
+class TestPoll:
+    async def test_handle_max_jobs_to_activate_when_no_jobs_to_activate(self, job_poller: JobPoller, caplog):
+        job_poller.max_task_count = 0
+
+        await job_poller.handle_max_jobs_to_activate()
+
+        assert re.search("Maximum number of jobs running for .*. Polling again in in 5 seconds...", caplog.text)
+
+    async def test_handle_max_jobs_to_activate_when_jobs_to_activate(self, job_poller: JobPoller, queue: asyncio.Queue, job_from_task: Job,
```

Comment:
```suggestion
    async def test_puts_job_in_queue_with_one_available_job(self, job_poller: JobPoller, queue: asyncio.Queue, job_from_task: Job,
```

Reply:
I have incorporated this suggestion, thanks!